fly/integration/sync_test.go

```diff
@@ -61,17 +61,9 @@ var _ = Describe("Syncing", func() {
 
 			flyVersion = fmt.Sprintf("%d.%d.%d", major, minor, patch+1)
 		})
-		It("downloads and replaces the currently running executable", func() {
```

Comment:
We still intend to support `fly -t foo sync` so I don't think we should switch the tests completely over to `-c`. New tests should be added instead.